### PR TITLE
Restructure app directory structure, improve performance, and fix detection bugs

### PR DESCRIPTION
**Pull request by RohanSeam**
# Summary
Restructure code base to adhere to a more intuitive model. Improved performance by removing unnecessary frame copies, fixed bugs around flickering of detection box and not showing up reliably. Also, cleaned up unnecessary code and comments. 

## What I did here
- Restructure of code repo

## How to test
- Run app on BR_DETECTION_CLEANUP

### CODE REVIEW

File: application/TFLite_detection_stream.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+output_details = None
+labels = None
+height = None
+width = None
+input_mean = 0 
+input_std = 0 
+boxes_idx = 0 
+classes_idx = 0
+scores_idx = 0
+
```
Comment (CurranSeam) [inline application/TFLite_detection_stream.py:33]: Technically if these are global variables, they should be uppercase. We can make a new story for this

File: application/TFLite_detection_stream.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    if object_name == 'person' and current_time >= (message_time + datetime.timedelta(minutes = 5)):
+        filepath = CWD_PATH + "/snapshot.jpeg"
+        cv2.imwrite(filepath, frame)
+        sms_service.send_message(list(CAMERAS.keys())[idx], current_time, FEED_URL, filepath)
+        message_time = current_time
+
+# Main function for performing detection and notifying users of activity
+def detection():
+    global frame_rate_calc, message_time, freq, labels, CAMERAS, interpreter, width, height, input_mean, input_std, input_details, output_details, boxes_idx, scores_idx, classes_idx, min_conf_threshold
+
```
Comment (CurranSeam) [inline application/TFLite_detection_stream.py:113]: Does detection work without having all these global references?

File: application/TFLite_detection_stream.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        # Grab frame from video stream
+        for stream in CAMERAS.keys():
+            pic = CAMERAS.get(stream)[0].read()
+            frames.append(pic)
+
+        for idx, f in enumerate(frames):
+            # Force only use driveway. This can be removed if we
+            # want to add detection on the other cameras.
+            if idx > 0:
+                break
```
Comment (CurranSeam) [inline application/TFLite_detection_stream.py:130]: We may want to change this to `> 2` in another story to run detection on 3 cams

File: driver.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    # First label is '???', which has to be removed.
+    if TFLite_detection_stream.labels[0] == '???':
+        del(TFLite_detection_stream.labels[0])
+
+    # Load the Tensorflow Lite model.
+    # If using Edge TPU, use special load_delegate argument
+    if use_TPU:
+        TFLite_detection_stream.interpreter = Interpreter(model_path=PATH_TO_CKPT,
+                                experimental_delegates=[load_delegate('libedgetpu.so.1.0')])
+        print(PATH_TO_CKPT)
```
Comment (CurranSeam) [inline driver.py:94]: print statement here is missed old code I think

**Review by CurranSeam — APPROVED** [review]
Phew so much better now 🚒 💦 🔥 🙀

**Comment by RohanSeam** [comment]
Story for changes in comments: https://trello.com/c/yQ86QRKT

**Pull request merged**